definitions/artifacts/azure-cognitive-service-language.json

```diff
@@ -0,0 +1,64 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "$md": {
+    "access": "public",
+    "name": "azure-cognitive-services"
+  },
+  "type": "object",
+  "title": "Azure Cognitive Services",
```

Comment:
```suggestion
  "title": "Azure Cognitive Services :: Language",
```

Reply:
For educational purposes, what does the `: :` do here in the title?